ngraph/core/src/op/depth_to_space.cpp

```diff
@@ -115,123 +114,18 @@ void op::DepthToSpace::validate_and_infer_types()
 bool op::DepthToSpace::evaluate(const HostTensorVector& outputs,
                                 const HostTensorVector& inputs) const
 {
-    const auto& data = inputs[0];
-    const auto& out = outputs[0];
-    const auto& out_shape = out->get_shape();
-    size_t elem_size = data->get_element_type().size();
-
-    if (data->get_partial_shape().is_dynamic())
+    if (inputs[0]->get_partial_shape().is_dynamic())
     {
         return false;
```

Comment:
should not such checks be located inside reference kernel? Looks like it needs to return bool as well.